trimmer_decorator.rb

```diff
@@ -1,11 +1,7 @@
-require_relative './base_decorator'
+require './decorator'
```

Comment:
- Kindly In order to include classes, modules, etc in other files you have to use require_relative or require, and when it is some dependency on locating in gem kindly use require `ex: require 'date'.` so here above  I suggest you use require_relative instead of require for decorator `eg: require_relative  './decorator'`  and it will make your code looks more professional   :+1: :+1: :+1: